assets/javascripts/discourse/components/discourse-reactions-reaction-post.hbs

```diff
@@ -0,0 +1,59 @@
+<div
+  class="user-stream-item item {{if this.moderatorAction 'moderatorAction'}}"
```

Comment:
Unsure if converting the `classNameBindings` to this is the correct usage here.

Reply:
Yeah since there's only one conditional class, I think this is sensible. If the logic was mega-complex then it could've been refactored into getter, but there's no need for that here

Reply:
you can use `{{concat-class "user-stream-item item" (if this.moderatorAction "moderatorAction")}}`

It avoids all the handling of white space